app/lib/project/MetapolatorProject.js

```diff
@@ -614,6 +616,38 @@ define([
         exportController.export();
     };
 
+    _p.exportZipInstance = function(masterName, instanceName, precision) {
```

Comment:
I placed it here because I intended it to be the entry point for the user interface to invoke exporting a ZIP UFO. In that sense, for me it makes sense that such feature should be available as a method of the MetapolatorProject module.